src/sonic-yang-mgmt/_sonic_yang_ext.py

```diff
@@ -0,0 +1,661 @@
+# This script is used as extension of sonic_yang class. It has methods of
```

Comment:
Is prefixing filename with '_' a convention for extension files?
If not, can we drop it?

Reply:
Sure, I can drop it.

Reply:
Can you please resolve this?

If you have any updates, please commit

Reply:
Resolved this.